tests/integration-tests/conftest.py

```diff
@@ -372,12 +373,39 @@ def _config_renderer(config_file="pcluster.config.yaml", **kwargs):
         env = Environment(loader=file_loader)
         rendered_template = env.get_template(config_file).render(**{**kwargs, **default_values})
         config_file_path.write_text(rendered_template)
-        inject_additional_config_settings(config_file_path, request, region)
+        if not config_file.endswith("image.config.yaml"):
+            inject_additional_config_settings(config_file_path, request, region)
+        else:
+            inject_additional_image_configs_settings(config_file_path, request)
         return config_file_path
 
     return _config_renderer
 
 
+def inject_additional_image_configs_settings(image_config, request):
+    with open(image_config) as conf_file:
+        config_content = yaml.load(conf_file, Loader=yaml.SafeLoader)
```

Comment:
is `safe_load` equal to `load(..., Loader=yaml.SafeLoader)`?

Reply:
yes, they are same